lib/datasource/terraform-module/index.ts

```diff
@@ -89,6 +89,10 @@ export class TerraformModuleDatasource extends TerraformDatasource {
     if (failureCodes.includes(err.code)) {
       throw new ExternalHostError(err);
     }
+    // istanbul ignore if
+    if (err.response?.statusCode === 503) {
```

Comment:
Why not add to failure codes above?

Reply:
I think status code is different to err code

Reply:
ok, you're right